### PR TITLE
feat(post): deis open roadmap and planning process

### PR DESCRIPTION
**Pull request by gabrtv**

### CODE REVIEW

File: _posts/2015-05-21-deis-open-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    content: Introducing the Deis Open Roadmap and Planning Process
+  - name: keywords
+    content: deis, docker, paas, open source, roadmap
+---
+
+Today I'm thrilled to introduce our new [Open Roadmap](http://docs.deis.io/en/latest/roadmap/roadmap/) and [Planning Process](http://docs.deis.io/en/latest/roadmap/planning/). This ensures every community member can help shape the future of Deis.
+
+As Deis has grown over the last few months, many in the community have looked for ways to influence the roadmap.  Until now, this has required submitting GitHub proposals, starting mailing list discussions or chatting with maintainers in IRC.  While all of these methods are still valid, we now have a formal planning process that results in a public roadmap.
+
+Starting now, Release Planning Meetings will be held on the first Thursday of every month, aligned with our [Release Schedule](http://docs.deis.io/en/latest/roadmap/release_schedule/).  Meetings will take place via a public Google Hangout with access coordinated via the #deis channel on Freenode.  We look forward to your participation!
```
Comment (bacongobbler) [inline _posts/2015-05-21-deis-open-roadmap.md]: s/our/the

**Comment by carmstrong** [comment]
LGTM!

**Comment by mboersma** [comment]
LGTM modulo @bacongobbler's pronoun comment.

**Comment by bacongobbler** [comment]
LGTM

**Pull request merged**